purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -362,14 +373,21 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
      * @param [skus] List of skus
      * @param [callback] Response callback
      */
-    @JvmSynthetic
-    internal fun getNonSubscriptionSkus(
+    fun getNonSubscriptionSkus(
         skus: List<String>,
         callback: GetStoreProductCallback
     ) {
         getSkus(skus.toSet(), ProductType.INAPP, callback)
     }
 
+    @Deprecated(
+        message = "SkuDetails replaced with StoreProduct and ProductChangeListener " +
+            "replaced with ProductChangeCallback",
+        replaceWith = ReplaceWith(
+            expression = "purchaseProduct(activity, StoreProduct, " +
+                "upgradeInfo, ProductChangeCallback)"
+        )
+    )
```

Comment:
Will move these deprecated functions to the bottom after reviewing so it's easier to review now